### PR TITLE
8684: Added test for associated service declarative in order summary

### PR DESCRIPTION
**Pull request by CallumBradburyBJSS**
Story: [AB#7692](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7692)
Task: [AB#8684](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8684)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs [hunk 1]
```diff
@@ -229,6 +229,16 @@ public void GivenThereIsNoAssociatedServiceInTheOrderButTheSectionIsComplete()
             SetOrderAssociatedServicesSectionToComplete();
         }
 
+        [Given(@"the supplier added to the order has an associated service with a declarative flat price")]
+        public void GivenTheSupplierAddedToTheOrderHasAnAssociatedServiceWithADeclarativeFlatPrice()
+        {
+            var order = (Order)Context["CreatedOrder"];
+            order.SupplierId = 100000;
+            order.SupplierName = "Really Kool Corporation";
+            order.Update(Test.ConnectionString);
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: Would it be worth finding this via the db's rather than hardcoded?

Comment (CallumBradburyBJSS) [reply src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: without a doubt, that'd be nicer, was trying to follow the pattern of the other tests though, that'd be tech debt I think

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: I'll be tackling that tech debt when the latest round of AC tests are complete, but getting a start on it with this PR would be handy

Comment (CallumBradburyBJSS) [reply src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: I think if I got a start on it with this PR, it'd essentially be doing the entire tech debt, and that seems like it should be its own task

Comment (talawson05) [reply src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: If I was to vote, I'd say I'd rather callum just followed what we/I've done, and you/I tackled this so to keep callum available for dev work. If he or any dev starts to refractor they'll never make it out this side of the August deadline

Comment (PhysicsJon) [reply src/OrderFormAcceptanceTests.Steps/Steps/AssociatedServices.cs:238]: righto, I'll approve the PR

**Pull request merged**